### ISSUE TITLE
guest wifi state is lost 

### ISSUE THREAD

**Issue by frimtec** [frimtec/compal-wifi-switch#1]
When switching wifi off an on again, the activation state of the guest wifi is lost.

**Comment by frimtec** [comment]
Requires extension of compal library https://github.com/ties/compal_CH7465LG_py to support fun 308.

Example requests:
* Enable Guest-Wifi:
```
fun: 308
wlInterface2g: 3
wlInterface5g: 3
wlEnable2g: 1
wlEnable5g: 1
wlSsid2g: <GUEST_SID-2g>
wlSsid5g: <GUEST_SID-5g>
wlHiden2g: 2
wlHiden5g: 2
wlRekey2g: 0
wlRekey5g: 0
wlSecurity2g: 8
wlSecurity5g: 8
wlPSkey2g: <GUEST_PW-2g>
wlPSkey5g: <GUEST_PW-5g>
wlWpaalg2g: 3
wlWpaal
```
* Disable Guest-Wifi:
```
fun: 308
wlInterface2g: 3
wlInterface5g: 3
wlEnable2g: 2
wlEnable5g: 2
wlSsid2g: <GUEST_SID-2g>
wlSsid5g: <GUEST_SID-5g>
wlHiden2g: 2
wlHiden5g: 2
wlRekey2g: 0
wlRekey5g: 0
wlSecurity2g: 8
wlSecurity5g: 8
wlPSkey2g: <GUEST_PW-2g>
wlPSkey5g: <GUEST_PW-5g>
wlWpaalg2g: 3
wlWpaalg5g: 3
```

**Comment by frimtec** [comment]
Implemented in release 0.2.0.
